### PR TITLE
update for patch method

### PR DESCRIPTION
**Pull request by doktordirk**

### CODE REVIEW

File: api_rest.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ export class MyViewModel {
 
 ### .find(resource, criteria[, options])
 
-Find one or multiple resources.
+FIND request for one or multiple resources.
```
Comment (RWOverdijk) [inline api_rest.md]: If anything, it's GET. But I don't think this has to be changed.

Comment (RWOverdijk) [inline api_rest.md]: By that I mean, the original sentence was good.

**Comment by RWOverdijk** [comment]
Except the one comment, a big +1 from me

File: api_rest.md (hunk 1 above)
Comment (doktordirk) [inline api_rest.md]: purpose was to mention the method in there somewhere. FIND was wrong obviously. how about 

`Find one or multiple resources. (GET request)`

**Comment by doktordirk** [comment]
ps: i've no rights. upto you to merge later

File: api_rest.md (hunk 1 above)
Comment (RWOverdijk) [inline api_rest.md]: That sounds good 👍

**Comment by RWOverdijk** [comment]
I don't mind merging, but I did give you permissions on the docs repositories

**Comment by doktordirk** [comment]
ah, hmm oh . right :smile:

**Comment by RWOverdijk** [comment]
👍

**Pull request merged**